File: JAVASCRIPT/vocali.js
var vocali = ['a', 'i', 'e', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
var lettera;
var i;

lettera = window.prompt('Inserire una lettera: ');

for (i = 0; i <= 10; i++)
{
    if (lettera == vocali[i])
    {
        console.log('La lettera inserita ' + lettera + 'è una vocale.');
        break;
    }
    else
    {
        console.log('La lettera inserita è una consonante.');
        break;
    }
}

File: PYTHON/vocali.py
vocali = {'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'}
lettera = input('Inserire una lettera: ')

if lettera in vocali:
    print(f'La lettera inserita {lettera} è una vocale.')

else:
    print(f'La lettera inserita {lettera} è una consonante.')